controller/api/destination/watcher/endpoints_watcher_test.go

```diff
@@ -2308,3 +2308,115 @@ status:
 
 	listener.ExpectRemoved([]string{"172.17.0.12:8989"}, t)
 }
+
+// Test that when an endpointslice gets a hint added, then mark it as a change
+func TestEndpointSliceAddHints(t *testing.T) {
+	k8sConfigsWithES := []string{`
+kind: APIResourceList
+apiVersion: v1
+groupVersion: discovery.k8s.io/v1
+resources:
+- name: endpointslices
+  singularName: endpointslice
+  namespaced: true
+  kind: EndpointSlice
+  verbs:
+    - delete
+    - deletecollection
+    - get
+    - list
+    - patch
+    - create
+    - update
+    - watch
+`, `
+apiVersion: v1
+kind: Service
+metadata:
+  name: name1
+  namespace: ns
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 8989`, `
+addressType: IPv4
+apiVersion: discovery.k8s.io/v1
+endpoints:
+- addresses:
+  - 172.17.0.12
+  conditions:
+  ready: true
+  targetRef:
+    kind: Pod
+    name: name1-1
+    namespace: ns
+  topology:
+  kubernetes.io/hostname: node-1
```

Comment:
Does this need an indent?

Reply:
I added one. It doesn't seem to affect anything here.